test/obb/events/lobby_events_test.clj

```diff
@@ -0,0 +1,43 @@
+(ns obb.events.lobby-events-test
+  (:use clojure.test)
+  (:require [clojure.core.async :as async]
+            [com.stuartsierra.component :as component]
+            [obb.models.game-info :as game-info]
+            [obb.events.lobby-events :as lobby-events]))
+
+(defn lobby-events-system
+  "Buils a specific test system with lobby events and it's dependencies"
+  []
+  (component/system-map
+    :lobby-events (lobby-events/create)))
+
+(deftest create-start-stop-component
+  (testing "creates a ready to use component"
```

Comment:
Does this test really test if the component is ready to use? Not sure what the intent of the test is, but I think it would make sense to change the description.

I get the impression this test is not needed anymore. While I can imagine why you used it while you were building the first version of the component, I think the basic operation of the component is now validated by other tests. If this is true, I think you should consider deleting it.